src/components/socialMedia/SocialMedia.js

```diff
@@ -1,23 +1,36 @@
 import React from "react";
 import "./SocialMedia.css";
 import { socialMediaLinks } from "../../portfolio";
+import styled from 'styled-components';
+
+const IconWrapper = styled.span`
+	i {
+		background-color: ${props => props.backgroundColor};
+	}
+	&:hover i {
+		background-color: black;
```

Comment:
Now since we have used `styled-components` and we can change the background color dynamically. Should we have `onHover` background color as `theme.text`. It will suite more

Reply:
That looks great! Just checked with all the themes🌈. Changes on the way🚀